packages/devtools_app/lib/src/screens/inspector_v2/inspector_screen.dart

```diff
@@ -358,15 +341,13 @@ class FlutterInspectorSettingsDialog extends StatelessWidget {
               gaItem: gac.inspectorHoverEvalMode,
             ),
             const SizedBox(height: denseSpacing),
-            const InspectorDefaultDetailsViewOption(),
-            const SizedBox(height: denseSpacing),
             ...dialogSubHeader(theme, 'Package Directories'),
             Row(
               mainAxisAlignment: MainAxisAlignment.spaceBetween,
               children: [
                 Expanded(
                   child: Text(
-                    'Widgets in these directories will show up in your summary tree.',
+                    'Widgets in these directories will show up in your widget tree.',
```

Comment:
"will be visible by default in your widget tree"? Or something to communicate how these widgets are differentiated in the single tree from implementation widgets?

Reply:
done!